domains/auth/Auth.API/Integration.Tests/AuthWebApplicationFactory.cs

```diff
@@ -43,16 +43,16 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
         {
             services.Remove(services.First(x => x.ServiceType == typeof(DbContextOptions<DataContext>)));
             services.Remove(services.First(x => x.ServiceType == typeof(DataContext)));
-            services.Remove(services.First(x => x.ServiceType == typeof(NpgsqlDataSourceBuilder)));
-            services.AddSingleton(new NpgsqlDataSourceBuilder(testContainer.GetConnectionString()));
-            services.AddDbContext<DataContext>((serviceProvider, options) => options.UseNpgsql(serviceProvider.GetRequiredService<NpgsqlDataSourceBuilder>().Build()));
+            services.AddDbContext<DataContext>(options => options
+                .UseNpgsql(DataContext.GenerateNpgsqlDataSource(testContainer.GetConnectionString()))
+                .ConfigureWarnings(warnings => warnings.Ignore(CoreEventId.ManyServiceProvidersCreatedWarning)));
```

Comment:
Why ignore the warnings here?

Reply:
The warnings it produces is a product of the upgrade to ef-core 8. Microsoft themselves suggests, that it can be ignored. AKA, it seems to be a false positive.